e2e/homepage.spec.ts

```diff
@@ -1,11 +1,49 @@
-import { expect, test } from '@playwright/test';
+import {
+	LOGIN_BUTTON,
+	THREE_BACKGROUND_CANVAS,
+	TOKENS_SKELETONS_INITIALIZED
+} from '$lib/constants/test-ids.constant';
+import { testWithII } from '@dfinity/internet-identity-playwright';
+import { expect, test, type Page } from '@playwright/test';
 
 const testUrl = '/';
 
-test('should display not logged in homepage', async ({ page }) => {
-	await page.goto(testUrl);
+const hideHeroAnimation = async (page: Page): Promise<void> => {
+	await page
+		.getByTestId(THREE_BACKGROUND_CANVAS)
+		.evaluate((element) => (element.style.display = 'none'));
+};
 
-	await page.getByTestId('login-button').waitFor();
+test.describe('logged out user', () => {
+	test.beforeEach(async ({ page }) => {
+		await page.goto(testUrl);
 
-	await expect(page).toHaveScreenshot();
+		await page.getByTestId(LOGIN_BUTTON).waitFor();
+	});
+
+	test('should display homepage in logged out state', async ({ page }) => {
+		await hideHeroAnimation(page);
+
+		await expect(page).toHaveScreenshot({ fullPage: true });
+	});
+});
+
+testWithII.describe('logged in user', () => {
+	testWithII.beforeEach(async ({ page, iiPage }) => {
+		const url = 'http://127.0.0.1:4943';
+		const canisterId = 'rdmx6-jaaaa-aaaaa-aaadq-cai';
```

Comment:
for another side note: I am thinking that in another PR we should put these values somewhere to be re-used throughout the code...

Reply:
Definitely. I will move all consts and helpers methods (`hideHeroAnimation` in this case) to separate files, especially since we'll proceed with one-spec-per-file approach and hence we'll need to re-use those things.

Reply:
Actually, sorry, the canisterId you could take it directly from `canister_e2e_ids.json` in this PR, right?

Reply:
Good tip, will use that function.

Reply:
I moved the constants/utils already in this PR. Though, for now, `getInternetIdentityCanisterId` returns a hardcoded string because in order to use `readCanisterIds` I need to update Dockerfile.e2e (at the moment, `canister_e2e_ids.json` is not a part of the e2e docker image). I will do it in a separate PR.